include/umf/memory_pool_ops.h

```diff
@@ -166,6 +169,24 @@ typedef struct umf_memory_pool_ops_t {
                             const char *name, void *arg, size_t size,
                             umf_ctl_query_type_t queryType, va_list args);
 
+    ///
```

Comment:
add a comment above "The following operations were added in ops version 1.1"

Reply:
Rebased.